client/src/components/Employees/EachEmployee.css

```diff
@@ -41,12 +41,105 @@
   padding-bottom: 5%;
 }
 
-.first {
+.moffitt3Display {
   flex: 1;
-  background-color: #1de0a6;
+  background-color: #ff8d06;
   color: white;
-  width: 90%;
+  margin-right: 60%;
+  height: 30px;
+  width: 140px;
+  padding: 2%;
+  margin: 20px;
+  font-family: "Open Sans", sans-serif;
+  letter-spacing: 1px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 30px;
+  text-align: center;
+  border-radius: 20.5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 0%;
+  padding-bottom: 0%;
+}
+.moffitt3NoDisplay {
+  flex: 1;
+  background-color: white;
+  color: white;
+  margin-right: 60%;
+  height: 30px;
+  width: 140px;
+  padding: 2%;
+  margin: 20px;
+  font-family: "Open Sans", sans-serif;
+  letter-spacing: 1px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 30px;
+  text-align: center;
+  border-radius: 20.5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 0%;
+  padding-bottom: 0%;
+}
+.moffitt4Display {
+  flex: 1;
+  background-color: #04b17e;
+  color: white;
+  margin-right: 60%;
+  height: 30px;
+  width: 140px;
+  padding: 2%;
+  margin: 20px;
+  font-family: "Open Sans", sans-serif;
+  letter-spacing: 1px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 30px;
+  text-align: center;
+  border-radius: 20.5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 0%;
+  padding-bottom: 0%;
+}
+.moffitt4NoDisplay {
```

Comment:
Instead of making the background colors white for all of the non-displaying library tags, let's just do `display: none` instead so that the tags all are equally spaced apart.

Reply:
yeet